providers/org/datacite/oai/apps.py

```diff
@@ -0,0 +1,10 @@
+from share.provider import OAIProviderAppConfig
```

Comment:
Is there any reason to keep the old OAI harvester around? It seems like the new namespace would be a superset of the oai namespace?